docs/manufacturing.md

```diff
@@ -11,7 +52,25 @@ Connectors' pinouts used for flashing can be found in the picture below.
 
 ![](figures/flashing_interface.svg)
 
-3) After flashing is done, all the PCBs should be stacked, aligned,
+To flash Babel-Babel, the Babel part of the board has to be flashed first then the DCP part will be flashed.
+To Flash Babel follow the procedure bellow:
+* Turn off one babel by toggling the switch
+* Connect 1 DCP and 1 Babel from PC
+  
+![](figures/flashing1.jpg)
+
+* Connect 1 CAN port from Babel to 1 port of Babel-Babel on the side that'll be flashed
+* Navigate to drwatson_babel repository and run "./drwatson_babel.py (babel port path)"
+ 
+![](figures/flashing3.png)
+
+* Enter provided credentials and press enter and the flashing process will start
+
+![](figures/flashing4.png)
+
+### To flash the DCP part follow [this link](https://github.com/Zubax/dronecode_probe/blob/master/MANUFACTURING.md)
```

Comment:
```suggestion
To flash the DCP part, follow [the DCP Manufacturing documentation](https://github.com/Zubax/dronecode_probe/blob/master/MANUFACTURING.md).
```

Please do not use constructs like "this link" because they lose meaning when the text is copied or printed on a non-rich medium.

Also, this should not be a header.